### DISCUSSION TITLE
Is it possible to close a book by clicking on a button?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#983] [Q&A]
Hi, I've started making books and objects, only I don't know the commands I can use, and I'd like to know if there is a command for buttons to close the object as "close" but integrated into the button. I tried with a requires_not_status that triggers once the button is activated but it doesn't work : 
![Capture d'écran 2024-12-31 162044](https://github.com/user-attachments/assets/5043c2d3-6939-4dda-a6ff-d390e8cb5a71)
![Capture d'écran 2024-12-31 162058](https://github.com/user-attachments/assets/030cb48c-7c01-4ea6-9993-d1f289a86dcd)
and my book : 
[acces1.txt](https://github.com/user-attachments/files/18282393/acces1.txt)

**Comment by dorkster** [comment]
I've made this possible in [1.14.81](https://github.com/flareteam/flare-engine/commit/1ec0bfaadbdcfb4b3029755b53b1f585c3d21ac9). To use it, just add `book=close` to the button event.

**Reply by BuzzLEquerre** [reply]
![Capture d'écran 2024-12-31 192455](https://github.com/user-attachments/assets/f7999841-559b-45a3-a26d-b299c93d246a)
This effectively disables the images and buttons but there is still the background of the book as well as the button close iT is normal ?

**Reply by dorkster** [reply]
Are you sure you're using engine 1.14.81? What you're seeing will happen if you try to open a book that doesn't exist while another book is already open (i.e. it's trying to open a book with the filename "close").

**Reply by BuzzLEquerre** [reply]
Idk if I use 1.14.81 engine, I downloaded flare engine for windows here : 
https://flarerpg.org/download/

And I saw that a version 1.15 was in development, would that mean that I should recreate my mod or will it be able to adapt to this version ?

**Reply by dorkster** [reply]
If you got the release version (aka the one on the website's Download page), you're currently using version v1.14 of the engine. You can see the engine version number on the title screen in the top-right.

So the fix for this problem will be in engine v1.15. We aim to maintain compatibility with existing mods, so you shouldn't have to alter anything for your mod to still work.

**Reply by BuzzLEquerre** [reply]
Oh that's great ! Thanks for your work :)
